components/ui/CompactFeature.vue

```diff
@@ -3,12 +3,12 @@
     <img
       v-if="icon"
       class="compact-feature__icon"
-      :src="`/images/icons/icon${icon}White@3x.png`"
+      :src="`/images/icons/${icon}.${fileType}`"
     >
-    <h2 v-if="title" class="compact-feature__title">
+    <h3 v-if="title" class="compact-feature__title copy__subtitle">
```

Comment:
If you use the class `copy__subtitle`, the font type change on small screens, and following the Figma design, the title is always `'productive-heading-02'`. It could be better to remove the `copy__subtitle` class and add the CSS we need on the `compact-feature__title` class